### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ashishgupta6**
New Relase Update 

Version: 3.1.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,6 +272,11 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
 
 
 ## Changelog
+### 3.1.1
+ - Issue resoved of dlopen failed: library "libintelligence.so" not found
+ - Fixed the 'Supported FPS ranges cannot be null' issue.
+ - Addressed the error: 'BioMetricStatus cannot be cast to a non-null type android.hardware.fingerprint.FingerprintManager.'
+ - Now, EventMetrics will be triggered only in case of failure."
```
Comment (anz3r) [inline README.md]: dont need to add this (last one)

File: README.md [hunk 2]
```diff
@@ -272,6 +272,11 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
 
 
 ## Changelog
+### 3.1.1
+ - Issue resoved of dlopen failed: library "libintelligence.so" not found
+ - Fixed the 'Supported FPS ranges cannot be null' issue.
+ - Addressed the error: 'BioMetricStatus cannot be cast to a non-null type android.hardware.fingerprint.FingerprintManager.'
```
Comment (anz3r) [inline README.md]: don't provide explicit paths. Capture just the gist of the issue

File: README.md [hunk 3]
```diff
@@ -272,6 +272,11 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
 
 
 ## Changelog
+### 3.1.1
+ - Issue resoved of dlopen failed: library "libintelligence.so" not found
```
Comment (anz3r) [inline README.md]: details not needed: first half is sufficient

File: README.md [hunk 4]
```diff
@@ -272,6 +272,11 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
...
+ - Fixed the 'Supported FPS ranges cannot be null' issue.
```
Comment (sreyans004) [inline README.md]: Please add point :

1. Fixed crash issue in android 6 & 7 devices.
2. Sign3 APIs now require minSdk 23 to run, apps targeting below SDK 23 will have to enclose Sign3 APIs within if condition.

**Review by sreyans004 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -272,6 +272,12 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
 
 
 ## Changelog
+### 3.1.1
+ - Fixed crash issue in android 6 & 7 devices.
+ - Fixed native code issue on some devices.
+ - Fixed camera supported FPS, BioMetricStatus issue.
+ - Now, EventMetrics will be triggered only in case of failure.
+ - Sign3 APIs now require a minimum SDK version of 23; apps targeting below this version must enclose Sign3 API calls within conditional checks.
```
Comment (anz3r) [inline README.md]: if we have added this, shouldn't this be a minor version bump instead of patch bump? 3.2 instead of 3.1.1

Comment (sreyans004) [reply README.md]: Yes, that would be better.

**Review by anz3r — APPROVED** [review]

**Pull request merged**